audio_transcription/backend/server.py

```diff
@@ -1,22 +1,32 @@
 from flask_ml.flask_ml_server import MLServer
-from flask_ml.flask_ml_server.models import ResponseBody
-from flask_ml.flask_ml_server.templates import FileML
+from flask_ml.flask_ml_server.models import ResponseBody, BatchTextResponse, TextResponse
+from flask_ml.flask_ml_server.templates.utils import response_body
+from flask_ml.flask_ml_server.templates.batchfile import FileInputs, create_task_schema_func
+from flask_ml.flask_ml_server.templates.no_parameters import NoParameters
 
 from ..ml.model import AudioTranscriptionModel
 
 model = AudioTranscriptionModel()
 server = MLServer(__name__)
-file_ml = FileML()
 
 
-@server.route("/transcribe", task_schema_func=file_ml.task_schema_func)
-def transcribe(inputs: file_ml.input_type, parameters: file_ml.parameter_type) -> ResponseBody:
+@server.route("/transcribe", task_schema_func=create_task_schema_func())
+def transcribe(inputs: FileInputs, parameters: NoParameters) -> ResponseBody:
     print("Inputs:", inputs)
     print("Parameters:", parameters)
     files = [e.path for e in inputs["file_inputs"].files]
     results = model.transcribe_batch(files)
     results = {r["file_path"]: r["result"] for r in results}
-    return file_ml.generate_text_response(results)
-
+    return response_body(
+        BatchTextResponse(
+            texts=[
+                TextResponse(
+                    title=k,
+                    text=v,
+                )
+                for k, v in results.items()
+            ]
+        )
+    )

```

Comment:
I think the BatchTextReponse can still be here, instead of inside the templates. Instead the `ResponseBody(root=...)` step can be replaced with `response_body(...)`

This:
1) Allows the templates to only be concerned with inputs and UI
2) Prevents another layer of abstraction (dictionary to BatchTextResponse) that keeps the API explainable to users.